### PR TITLE
Users/Tutorial.rst: Output examples added

### PR DESCRIPTION
**Pull request by adeora7**
Users/Tutorial.rst: Output examples added

Added the examples of output when a user runs coala
on coala-tutorial code.

Closes https://github.com/coala/coala/issues/4254

### CODE REVIEW

**Comment by adeora7** [comment]
@Makman2  updated to change to lowercase coala. But could you please tell me why it is lowercase? :)

**Comment by Makman2** [comment]
The name "coala" is just written lowercase. When you use names you keep its spelling (including upper- and lowercasing) :+1:

File: Users/Tutorial.rst [hunk 1]
```diff
@@ -66,12 +66,57 @@ coala will now ask you for missing values that are needed to perform the
 analysis, which in this case is only the ``use_spaces`` setting. We
 recommend setting it to ``True``.
 
+::
+
+    Please enter a value for the setting "use_spaces" (True if spaces
+    are to be used instead of tabs.) needed by SpaceConsistencyBear
+    for section "cli"
+
+We recomment setting it to ``True``.
```
Comment (Makman2) [inline Users/Tutorial.rst]: --> recommend
However, this is duplicated from above, please remove this sentence.

File: Users/Tutorial.rst [hunk 2]
```diff
@@ -66,12 +66,57 @@ coala will now ask you for missing values that are needed to perform the
...
+    |   4|   4|     printf("Welcome to coala. Keep following the
+                    tutorial, you are doing a great job so far!\n");
+    |    | *0: Do nothing
+    |    |  1: Open file(s)
+    |    |  2: Apply patch
+    |    |  3: Print more info
+    |    |  4: Add ignore comment
+    |    | Enter number (Ctrl-D to exit): 2
+
+If the patch was applied succesfully, you should see something like this.
```
Comment (Makman2) [inline Users/Tutorial.rst]: colon instead of period

**Comment by adeora7** [comment]
@Makman2 updated as per comments :)

**Comment by Makman2** [comment]
Looks fine for me :+1:

**Comment by Makman2** [comment]
ack 9f7b8c9

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**